test/integration/manager-discovery.yaml.tmpl

```diff
@@ -27,9 +27,14 @@ spec:
       - name: manager
         image: {{.hub}}/runtime:{{.tag}}
         imagePullPolicy: Always
-        args: ["discovery", "-v", "2", "-n", "{{.namespace}}"]
+        args: ["discovery", "-v", "2", "-n", "$(POD_NAMESPACE)"]
         ports:
         - containerPort: 8080
+        env:
```

Comment:
please remove this. discovery can be run in a separate namespace, and using downward API here is not appropriate.